wsrtc.js

```diff
@@ -4,13 +4,24 @@ import {ensureAudioContext, getAudioContext} from './ws-audio-context.js';
 import {encodeMessage, encodeAudioMessage, encodePoseMessage, encodeTypedMessage, decodeTypedMessage, getEncodedAudioChunkBuffer, getAudioDataBuffer/*, loadState*/} from './ws-util.js';
 import * as Z from 'zjs';
 
+import metaversefileApi from 'metaversefile';

```

Comment:
Don't use `metaversefile` here. wsrtc should be more independent.
Emit the message/data you need from the connection socket, and then keep the player logic in `app`.